analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -312,7 +317,12 @@ class Pip(
                 // The tree does not contain a node for the project itself. Its dependencies are on the root level
                 // together with the dependencies of pipdeptree itself, which we need to filter out.
                 fullDependencyTree.filterNot {
-                    it["package_name"].textValue() in PIPDEPTREE_DEPENDENCIES
+                    val packageName = it["package_name"].textValue()
+                    val packageVersion = it["required_version"].textValueOrEmpty()
```

Comment:
Shouldn't this rather use the "installed_version" key?

Reply:
good catch!